lib/PullRequestAction.ts

```diff
@@ -0,0 +1,29 @@
+import { OctokitAction } from './OctokitAction';
+
+export abstract class PullRequestAction extends OctokitAction {
+  protected abstract processJiraIssue(issueId: string): Promise<void>;
+
+  protected async execute(): Promise<void> {
+    const issueIds = await this.getLinkedJiraIssues();
+
+    if (issueIds.length === 0) {
+      console.warn('No JIRA issue found in the PR title.');
+    }
+
+    for (const issueId of issueIds) {
+      await this.processJiraIssue(issueId);
+    }
+  }
+
+  private async getLinkedJiraIssues(): Promise<string[]> {
```

Comment:
No `get`, name inspired by old implementation, being explicit that what ever is in title is meant to be a fixed issue
```suggestion
  private async fixedJiraIssues(): Promise<string[]> {
```